src/common/marlin_vars.h

```diff
@@ -29,7 +29,11 @@
 #define MARLIN_VAR_SD_PRINT 0x15 // R:  uint8, card.flag.sdprinting
 #define MARLIN_VAR_SD_PDONE 0x16 // R:  uint8, card.percentDone()
 #define MARLIN_VAR_DURATION 0x17 // R:  uint32, print_job_timer.duration()
-#define MARLIN_VAR_MAX      MARLIN_VAR_DURATION
+#define MARLIN_VAR_MEDIAINS 0x18 // R:  uint8, media_is_inserted()
+#define MARLIN_VAR_PRNSTATE 0x19 // R:  marlin_print_state_t, marlin_server.print_state
+#define MARLIN_VAR_FILENAME 0x1a // R:  char*,
+#define MARLIN_VAR_FILEPATH 0x1b // R:  char*,
+#define MARLIN_VAR_MAX      MARLIN_VAR_FILEPATH
 
```

Comment:
`static_assert(MARLIN_VAR_MAX < 64, "Too many variables");`

Reply:
there is even a non-standard static assert for plain C in gcc

Reply:
commit 7daf3c4